.github/workflows/pr-review-notify.yml

```
@@ -0,0 +1,122 @@
+
+jobs:
+  notify-review-comment:
+    runs-on: ubuntu-latest
+    # issue_comment は PR 以外(Issue)でも発火するため PR のみフィルタ
+    if: >-
+      (github.event_name != 'issue_comment' || github.event.issue.pull_request) &&
+      (github.event_name != 'check_run' || github.event.check_run.conclusion == 'failure')
+    steps:
+      - name: Extract comment info
+        id: extract
+        run: |
```

actionlint (1.7.11)

[error] "github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)